Lombiq.ChartJs.Samples/Constants/ContentItemIds.cs

```diff
@@ -0,0 +1,6 @@
+namespace Lombiq.ChartJs.Samples.Constants;
+public static class ContentItemIds
+{
+    public const string IncomeTagsTaxonomy = "4d3c8qehr0pepzrext3qsk2enn";
+    public const string ExpenseTagsTaxonomy = "4r2zgez4rs3dc3vqp332yrcdn3";
```

Comment:
Could you replace the random IDs with human readable ones like this?
```suggestion
    public const string IncomeTagsTaxonomy = "taxonomyincometags00000000";
    public const string ExpenseTagsTaxonomy = "taxonomyexpensetags0000000";
```

I'm on a crusade against random content item IDs, because they make the recipes hard to read and even harder to debug. We have applied this practice in SNOW and it's definitely cleaner. please do a replace all for these two IDs.